src/client/index.js

```diff
@@ -68,7 +71,12 @@ class Hammerhead {
             event:    eventUtils,
             position: positionUtils,
             style:    styleUtils,
-            types:    typeUtils
+            types:    typeUtils,
+            service:  {
+                isJQueryObj: isJQueryObj,
+                extend:      extend,
+                trim:        trim
+            }
```

Comment:
Maybe we should add methods to the utils root instead of creating a new `service` entity?